CHANGELOG.md

```diff
@@ -1,24 +1,2 @@
-## 1.0.4
- - Update to Elasticsearch 1.7
-
-## 1.0.3
- - Add HTTP proxy support 
-
-## 1.0.2
- - Upgrade Manticore HTTP Client
-
-## 1.0.1
- - Allow client certificates
-
-## 0.2.9
- - Add 'path' parameter for ES HTTP hosts behind a proxy on a subpath
-
-## 0.2.8 (June 12, 2015)
- - Add option to enable and disable SSL certificate verification during handshake (#160)
- - Doc improvements for clarifying round robin behavior using hosts config
-
-## 0.2.7 (May 28, 2015)
- - Bump es-ruby version to 1.0.10
-
-## 0.2.6 (May 28, 2015)
- - Disable timeouts when using http protocol which would cause bulk requests to fail (#103)
+## 2.0.0
+  - Initial Release. Only supports Node/Transport in ES2.x
```

Comment:
oh duh I'm silly. THis is a new plugin basically, so a new changelog makes sense.